Kernel/TimerQueue.cpp

```diff
@@ -79,7 +79,7 @@ TimerId TimerQueue::add_timer(NonnullRefPtr<Timer>&& timer)
 
     timer->m_id = ++m_timer_id_count;
     VERIFY(timer->m_id != 0); // wrapped
-    add_timer_locked(move(timer));
+    add_timer_locked(timer);
```

Comment:
Was this supposed to be in this PR?

Reply:
Yes, this was intentional as this method wasn't used by anything else, and contained a bug (it tries to return timer->m_id after moving the value out from timer, which results in a nullptr VERIFY being triggered). After looking at it again, I think it would actually be better to store the timer_id in a local variable and return that instead, I'll change it to that in the next revision.

Reply:
if youre fixing a bug in a separate subsystem a separate commit would be nice :)

Reply:
Yep, I've moved it into another commit.